cmd/controller/app/options/options.go

```diff
@@ -285,6 +295,18 @@ func (o *ControllerOptions) Validate() error {
 		return fmt.Errorf("invalid default issuer kind: %v", o.DefaultIssuerKind)
 	}
 
+	if o.KubernetesAPIBurst <= 0 {
+		return fmt.Errorf("invalid value for kube-api-burst: %v must be higher than 0", o.KubernetesAPIBurst)
```

Comment:
Is there a way to make it unlimited? `0` or `-1` or something else?

Reply:
Not supported by client-go... 0 is equal to the default value